ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -317,55 +318,51 @@ export default defineComponent({
         last: props.editHomeOwner?.individualName.last || ''
       }
     } else {
-      defaultHomeOwner.organizationName =
-        props.editHomeOwner?.organizationName || ''
+      defaultHomeOwner.organizationName = props.editHomeOwner?.organizationName || ''
     }
 
-    const localState = reactive({
-      group: getGroupForOwner(
-        props.editHomeOwner?.id
-      ) as MhrRegistrationHomeOwnerGroupIF,
-      ownersGroupId: computed(() =>
-        showGroups.value ? localState.group?.groupId : null
-      ),
-      owner: { ...defaultHomeOwner },
-      ownerGroupId: undefined,
-      showGroups: showGroups,
-      isPerson: props.isHomeOwnerPerson,
-      isAddingHomeOwner: props.editHomeOwner == null,
-      fraction: {
+    const allFractionalData = (getMhrRegistrationHomeOwnerGroups.value || [{}]).map(group => {
+      return {
+        groupId: group.groupId || '1',
+        type: group?.type || '',
+        interest: group?.interest || '',
+        interestNumerator: group?.interestNumerator || null,
+        interestTotal: group?.interestTotal || null,
+        tenancySpecified: group?.tenancySpecified || null
+      }
+    })
+
+    const hasMultipleOwnersInGroup =
```

Comment:
Do have some types (interfaces) we can use/create for these properties. Might be useful for other devs.

Reply:
Yea, I do have an interface, just this obj has an extra field (groupId). I'll add a type, thanks!

Reply:
Added interface.